go/porcelain/deploy_containment_test.go

```
@@ -0,0 +1,116 @@
+	assert.NotNil(t, files.Files["index.html"], "regular file should be deployed")
+	assert.Nil(t, files.Files["leak"], "symlink should not be deployed")
+	for name := range files.Files {
+		assert.NotEqual(t, "out-of-tree", contentOf(t, deployDir, name))
+	}
+}
+
+func contentOf(t *testing.T, dir, name string) string {
+	t.Helper()
+	b, err := os.ReadFile(filepath.Join(dir, filepath.FromSlash(name)))
+	require.NoError(t, err)
+	return fmt.Sprintf("%s", b)
```

golangci-lint (2.12.2)

[error] S1025: the argument's underlying type is a slice of bytes, should use a simple conversion instead of fmt.Sprintf

(staticcheck)